tests/lib/snaps/test-snapd-tools-core18/meta/snap.yaml

```diff
@@ -0,0 +1,30 @@
+name: test-snapd-tools-core18
+version: 1.0
+architectures: ["all"]
+base: core18
```

Comment:
To avoid the duplication it would be nice to have a command like `install_local` that renames a snap (if needed) and adds the base declaration to it. This can be a follow up but I'm worried we will diverge on the copied snaps.

Reply:
Yeah, that is a good point, I add a note for a followup.